### PR TITLE
Added ability to get number of ignored tests and total number of tests.

### PR DESCRIPTION
**Pull request by trianglee**

### CODE REVIEW

**Comment by trianglee** [comment]
We are running Unity in a few different setups, some of them make it very hard to mechanically parse the tests output.
It is possible to get the logs for a human to browse if things go wrong, but this is a fairly painful process, and the resulting log is often hard to parse due to "noise" - the logging mechanism adds timestamps and other indicators that we can't disable.
To overcome this, we wrap the call to Unity with some code that sends out the vital statistics (successful tests, failed tests, ignored tests) to the outside world.

**Comment by mvandervoord** [comment]
Makes sense to me. Thanks for elaborating!  :)
